src/replication/intro.rst

```diff
@@ -26,18 +26,46 @@ the end of the process, all active documents on the source database are also in
 the destination database and all documents that were deleted in the source
 databases are also deleted on the destination database (if they even existed).
 
-Triggering Replication
-======================
 
-Replication is controlled through documents in the
-:ref:`_replicator <replicator>` database, where each document describes one
-replication process (see :ref:`replication-settings`).
+Transient and Persistant Replication
+====================================
+
+There are two different kinds of setting up a replication. The first one that was
+introduced into CouchDB leads do a replication that could be called `trancient`.
+Trancient means that there are no documents backing up the replication. So after a
+restart of the CouchDB server the replication will disapear. Later in time the
+:ref:`_replicator <replicator>` database was introduced which keeps documents
+containing your replication parameter. Such a replication can be called `persisent`.
+Trancient and persistent replications can have different
+:ref:`replication states <replicator/states>`.
+
+The reason there are two is because transient ones where implemented first and
+were kept for backwards compatibility and are actually useful in some cases when
+programmatically creating replication jobs.
```

Comment:
Can this paragraph be eliminated? It's mostly redundant from the previous paragraph. Maybe above just add a note that the transient replications were preserved for backward compatibility.